scripts/upgrade

```diff
@@ -31,6 +31,7 @@ upstream_version=$(ynh_app_upstream_version)
 #=================================================
 
 domain=$(ynh_app_setting_get $app special_domain)
+server_name=$(ynh_app_setting_get $app server_name)
```

Comment:
You need to support the retro compatibility. So in the case of the `server_name` is not defined, you need to define it.

You probably need to add a code like that:

```bash
if [ -z $server_name ]; then
   server_name=$domain
   ynh_app_setting_set $app server_name  $domain
fi